src/GafferTest/MultiplyNode.cpp

```diff
@@ -45,13 +45,15 @@ GAFFER_GRAPHCOMPONENT_DEFINE_TYPE( MultiplyNode )
 
 size_t MultiplyNode::g_firstPlugIndex = 0;
 
-MultiplyNode::MultiplyNode( const std::string &name )
+MultiplyNode::MultiplyNode( const std::string &name, bool brokenAffects )
 	:	ComputeNode( name )
 {
 	storeIndexOfNextChild( g_firstPlugIndex );
 	addChild( new IntPlug( "op1" ) );
 	addChild( new IntPlug( "op2" ) );
 	addChild( new IntPlug( "product", Plug::Out ) );
+
+	m_brokenAffects = brokenAffects;
```

Comment:
Better to use an initialiser than an assignment in the constructor.

Reply:
Addressed in dd25539